app/index.html

```diff
@@ -34,6 +34,10 @@ <h3 class="hero-title">
           <a class="col-11 col-md-5 btn btn-outline-primary btn-lg shadow ml-md-3 mt-3 mt-md-0" href="/$( _lang_ )$/how-to-pledge">$( index.contribute )$</a>
         </div>
 
+        <div class="text-center mt-4">
+          <a href="/$( _lang_ )$/contribute-code" class="small">Contribute code and earn Bitcoin Cash<i data-feather="chevron-right" class="mb-1 ml-1"></i></a>
```

Comment:
This needs to be in i18n too. My bad, I'll open a PR fixing this later on.

Reply:
Ok. It's merged now. Can I help with the l18n?

Reply:
Sorry for the delay on that. I pushed the changes to master